docs/reference/setup/install/docker.asciidoc

```diff
@@ -134,6 +81,8 @@ endif::[]
 
 ifeval::["{release-state}"!="unreleased"]
 
+To bring up the cluster with  <<docker-prod-cluster-composefile,`docker-compose.yml`>>, run:
```

Comment:
```suggestion
To bring up the cluster in <<docker-prod-cluster-composefile,`docker-compose.yml`>>, run:
```

Maybe?

Reply:
Leaving as-is. I think putting the emphasis on using the compose file to bring up the cluster makes (slightly) more sense.